docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,71 @@
-- **Value**: The quantity of Wei or Ether that the sender account wishes to transmit to the recipient is represented by the value.
-- **Data**: If the transaction receiver is a smart contract, the data contains information for the contract's functions to be executed. This comprises data with varying lengths.
-- **Signature**: A signature indicates who sent the communication. The signature is created when an externally owned account confirms and signs the transaction with its private key.
+## Nature of transactions
 
-### Transaction Types
+There are various different ways of transacting on the Ethereum blockchain:
 
-- Simple or asset transfers: This refers to the regular tokens transfer in the form of ether from one account to another.
+- Regular transactions: A simple transaction from one account to another such as transferring a token.
+- Contract deployment transactions: A transaction with no `to` address and the `data` field contains the contract code.  
+- Contract deployment on zkSync Era is different from Ethereum.
+  - Ethereum: Contract deployment occurs when a user sends a transaction to the zero address `(0x000...000)` where the `data` field contains the contract bytecode concatenated with constructor parameters.
+  - zkSync Era: To deploy a contract on zkSync, a user calls the `create` function of the [ContractDeployer](../system-contracts.md#contractdeployer) and provides the hash of the contract to be published, as well as the constructor arguments. The contract bytecode is added to the `factory_deps` field of [EIP-712 transaction types](#eip-712-0x71). If the contract is a factory (i.e. it can deploy other contracts), the bytecode of all child contracts should be included in `factory_deps`.
+- Find out more about [contract deployment](../../building-on-zksync/contracts/contract-development.md) on zkSync Era.
+- Contract execution: A transaction that interacts with an already deployed smart contract where the `to` address is the smart contract's address.
 
-- Contract deployment transactions: Contract deployment on zkSync is quite different from Ethereum.
-  - Ethereum: Contract deployment occurs when a user sends a transaction to the zero address `(0x000...000)` with the `data` field of the transaction equal to the contract bytecode concatenated with the constructor parameters.
-  - zkSync Era: To deploy a contract on zkSync, a user calls the `create` function of the [ContractDeployer](../system-contracts.md#contractdeployer) and provides the hash of the contract to be published, as well as the constructor arguments. The contract bytecode itself is supplied in the `factory_deps` field of the EIP712 transactions. If the contract is a factory (i.e. it can deploy other contracts), the contracts bytecodes should be included in the `factory_deps`.
-  Read more on [contract deployment](../../building-on-zksync/contracts/contract-development.md).
+## Transaction types
```

Comment:
Let's add that this is all type of Eth txes

Reply:
except it's not because this one is zkSync specific: ### Priority: `0xff`

Reply:
So something like "For compatibility, most of zkSync Era transaction types are similar to Ethereum's".

Reply:
updated